crates/common_enums/src/enums.rs

```diff
@@ -37,3 +38,2097 @@ pub enum CountryCode {
+            630 => Self::PuertoRico,
+            634 => Self::Qatar,
+            638 => Self::Réunion,
+            642 => Self::Romania,
+            643 => Self::RussianFederation,
+            646 => Self::Rwanda,
+            652 => Self::SaintBarthélemy,
+            654 => Self::SaintHelenaAscensionandTristandaCunha,
+            659 => Self::SaintKittsandNevis,
+            662 => Self::SaintLucia,
+            663 => Self::SaintMartinFrenchpart,
+            666 => Self::SaintPierreandMiquelon,
+            670 => Self::SaintVincentandtheGrenadines,
+            882 => Self::Samoa,
+            674 => Self::SanMarino,
+            678 => Self::SaoTomeandPrincipe,
+            682 => Self::SaudiArabia,
+            686 => Self::Senegal,
+            688 => Self::Serbia,
+            690 => Self::Seychelles,
+            694 => Self::SierraLeone,
+            702 => Self::Singapore,
+            534 => Self::SintMaartenDutchpart,
+            703 => Self::Slovakia,
+            705 => Self::Slovenia,
+            90 => Self::SolomonIslands,
+            706 => Self::Somalia,
+            710 => Self::SouthAfrica,
+            239 => Self::SouthGeorgiaandtheSouthSandwichIslands,
+            728 => Self::SouthSudan,
+            724 => Self::Spain,
+            144 => Self::SriLanka,
+            729 => Self::Sudan,
+            740 => Self::Suriname,
+            744 => Self::SvalbardandJanMayen,
+            748 => Self::Swaziland,
+            752 => Self::Sweden,
+            756 => Self::Switzerland,
+            760 => Self::SyrianArabRepublic,
+            158 => Self::TaiwanProvinceofChina,
+            762 => Self::Tajikistan,
+            834 => Self::TanzaniaUnitedRepublicof,
+            764 => Self::Thailand,
+            626 => Self::TimorLeste,
+            768 => Self::Togo,
+            772 => Self::Tokelau,
+            776 => Self::Tonga,
+            780 => Self::TrinidadandTobago,
+            788 => Self::Tunisia,
+            792 => Self::Turkey,
+            795 => Self::Turkmenistan,
+            796 => Self::TurksandCaicosIslands,
+            798 => Self::Tuvalu,
+            800 => Self::Uganda,
+            804 => Self::Ukraine,
+            784 => Self::UnitedArabEmirates,
+            826 => Self::UnitedKingdomofGreatBritainandNorthernIreland,
+            840 => Self::UnitedStatesofAmerica,
+            581 => Self::UnitedStatesMinorOutlyingIslands,
+            858 => Self::Uruguay,
+            860 => Self::Uzbekistan,
+            548 => Self::Vanuatu,
+            862 => Self::VenezuelaBolivarianRepublicof,
+            704 => Self::VietNam,
+            92 => Self::VirginIslandsBritish,
+            850 => Self::VirginIslandsUS,
+            876 => Self::WallisandFutuna,
+            732 => Self::WesternSahara,
+            887 => Self::Yemen,
+            894 => Self::Zambia,
+            716 => Self::Zimbabwe,
+            _ => {
+                panic!()
+            }
```

Comment:
We can convert this function to return a `Result` instead and error out if an unknown country code is provided. But panicking and terminating a payments related system only because of an invalid input would not be a good experience for the users. They'd have no idea if their payments actually went through, for example.

Reply:
Yes, please remove the `panic!`, as suggested return a `Result`